pkg/unload/create.go

```diff
@@ -67,9 +68,9 @@ func (r *UnloadReconciler) unloadInternal(ctx UnloadReconcilerReqCtx) {
 	wg.Wait()
 }
 
-func removeAllPagesFromOneWorkerPod(podName string, ctx UnloadReconcilerReqCtx, clientSet *kubernetes.Clientset) {
+func removeAllPagesFromOneWorkerPod(podName string, ctx *UnloadReconcilerReqCtx, clientSet *kubernetes.Clientset) {
 	// Best effort to remove cached pages
-	paths := strings.Split(ctx.AlluxioCluster.Spec.Pagestore.HostPath, ",")
+	paths := strings.Split(*ctx.AlluxioClusterer.PagestoreSpec().HostPath, ",")
```

Comment:
check for nil before derefencing

Reply:
Do you recommend still checking if we give a default value, so it will never be nil?

Reply:
where's the default value set? is it external as in part of the yaml or internal as part of the golang code?